### ISSUE TITLE
Improve Application Gateways use a minimum TLS 1.2 doc

### ISSUE THREAD

**Issue by kislow** [Azure/PSRule.Rules.Azure#2063]
**What is the path or URL of the page you're reporting?**

[Azure Application Gateway SSL Policy](https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.AppGw.SSLPolicy/)

**How could the page be improved?**

Improve example code and a more detailed description. 

**Additional context**

The following error messages will occur:

```sh
# when no policyType is set
{"code": "ApplicationGatewaySslPolicyPropertyPolicyTypeRequired", "message": "Property 'PolicyType' is required for 'SslPolicy' for Application Gateway..."}
```
```sh
# when specifying policyType and minProtocolVersion together
{"code": "ApplicationGatewaySslPolicyPropertyNotSupportedForPolicyType", "message": "Specified PolicyType 'Predefined' does not support property 'MinProtocolVersion' for 'SslPolicy' for Application Gateway...}
```

**Comment by BernieWhite** [comment]
@kislow Good call out.
